app/script/entity/User.coffee

```diff
@@ -49,6 +49,7 @@ class z.entity.User
   ###
   constructor: (@id = '') ->
     @is_me = false
+    @is_bot = ko.observable false
```

Comment:
Good point. Wanted it to be a computed first but haven't figured out a way to detect this.